Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -0,0 +1,180 @@
+\chapter{Introduction}
+\setheader{Introduction}
+This document provides a sketch of the system that is going to be built during the context project multimedia services. The architecture of the system is explained in the form of high level components of the system. These components are split in to sub components and sub-systems.
```

Comment:
"Provides a sketch of the system"
I'd call this a high level overview of the system